openedx/features/course_experience/static/course_experience/js/CoursetalkReviews.js

```diff
@@ -0,0 +1,28 @@
+/*
+  Enable users to switch between viewing and writing CourseTalk reviews.
+ */
+
+export class CourseTalkReviews {  // eslint-disable-line import/prefer-default-export
+  constructor() {
+    const $courseTalkToggleReadWriteReviews = $('.toggle-read-write-reviews');
+    const $courseTalkScriptObject = $('.coursetalk-reviews-script');
+
+    const readSrc = '//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-read-reviews.js';
```

Comment:
Would you prefer it in the common.py settings file? I almost feel that a django config would be overkill if we only use it in this one location - especially because it is explicitly in the CourseTalk specific javascript file. Perhaps we could duplicate the file from the CDN and reference it directly as opposed to using the remote instance.